packages/augur-sdk/src/state/db/MarketDB.ts

```diff
@@ -158,33 +151,46 @@ export class MarketDB extends DerivedDB {
   }
 
   async getOrderBook(marketData: MarketData, numOutcomes: number, estimatedTradeGasCostInAttoDai: BigNumber): Promise<OrderBook> {
-    const currentOrdersResponse = await this.stateDB.CurrentOrders.where("[market+open]").equals([marketData.market, 1]).and((order) => {
-      return order.amount > '0x00';
-    }).toArray();
+    const currentOrdersResponse = await this.stateDB.ZeroXOrders
```

Comment:
Why aren't we using an index anymore? This does a full DB pull for every market on every 0x order to come in currently

Reply:
Yeah my mistake! - fixing